### PR TITLE
Adds some new wheelchair types

### LINKED ISSUE #7735: Wheelchair overlay issues
#### Brief description of the issue
Facing left or right with a wheelchair, your sprite overlays the wheelchair armrest/wheel sprite
#### What you expected to happen
for the armrest/wheel sprite to be over the character sprite, like it is when facing north
#### What actually happened
the character sprite being over the side wheelchair sprites
#### Steps to reproduce
get in a wheelchair
face left or right
#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

### PR DESCRIPTION
**Pull request by Greenjoe12345**
Adds a small electric wheelchair, designed for teshari. Also moves wheelchairs to their own .dmi file. It also adjusts how wheelchair folding and unfolding works to allow for subtypes of wheelchairs in the first place.
also adds a human-sized electric wheelchair, and fixes #7735 
note that these are just visually different, they don't act any differently from the normal wheelchair, other then the small being too small for humans and the such.

### CODE REVIEW

**Comment by Schnayy** [comment]
Please add the motorized wheelchair for humans as discussed in the discord alongside the teshari one, please. It doesn't make sense to only have them available to tesh.

**Review by Schnayy — CHANGES_REQUESTED** [review]
Marking for change requested above: please add a proper human-use electric wheelchair as well.

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 1]
```diff
@@ -206,13 +225,25 @@
...
+	icon = 'icons/obj/wheelchair.dmi'
+	icon_state = "teshchair_folded"
+	item_state = "teshchair"
+	w_class = ITEMSIZE_HUGE // Can't be put in backpacks. Oh well.
+	var/wheelchairtype = /obj/structure/bed/chair/wheelchair/smallmotor
+	var/wheelchairitemtype = /obj/item/smallmotorwheelchair
 
-/obj/item/wheelchair/attack_self(mob/user)
-		var/obj/structure/bed/chair/wheelchair/R = new /obj/structure/bed/chair/wheelchair(user.loc)
+/obj/item/smallmotorwheelchair/attack_self(mob/user)
```
Comment (Atermonera) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:245]: Should this path be changed? It looks like you refactored the changes to be on the base wheelchair type.

**Review by Atermonera — COMMENTED** [review]
I don't see anything restricting other mobs from buckling to the chair. What happens if a human (Or anything else) tries to use it?

Also, general interest check because it's been a couple of weeks, do you plan on addressing Schnayy's request?

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm (hunk 1 above)
Comment (Greenjoe12345) [reply code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:245]: That's being fixed on the next commit

**Comment by Greenjoe12345** [comment]
> I don't see anything restricting other mobs from buckling to the chair. What happens if a human (Or anything else) tries to use it?
> 
> Also, general interest check because it's been a couple of weeks, do you plan on addressing Schnayy's request?

I've got it in code now, but I just gotta sort a folded sprite

**Review by MistakeNot4892 — COMMENTED** [review]
I'm biased as I contributed code to this PR, but as far as I can tell it works locally.

**Review by Schnayy — APPROVED** [review]

**Pull request merged**